manifest.php

```diff
@@ -25,7 +25,7 @@
     'label' => 'TAO LTI Consumer',
     'description' => 'TAO LTI Consumer extension',
     'license' => 'GPL-2.0',
-    'version' => '0.2.0',
+    'version' => '0.3.0',
     'author' => 'Open Assessment Technologies SA',
     'requires' => [
         'taoLti' => '>=9.2.1',
```

Comment:
You should update this dependency I think.

Reply:
Correct. I forget this all the time...